content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -5,7 +5,7 @@ reviewers:
 - liggitt
 title: Using RBAC Authorization
 content_template: templates/concept
-aliases: [../../../rbac/]
+aliases: [/rbac/]
 weight: 70
 ---
 
```

Comment:
Looks good to me. We could add a comment (Hugo shortcode) here to remind people to update the page alias when localizing (lest they break the site).

Reply:
Absolutely - I'll add an issue on that front. Thank you @sftim and @kbhawkey for shepherding this PR 😄